Formula/arduino-cli.rb

```diff
@@ -4,14 +4,15 @@ class ArduinoCli < Formula
   url "https://github.com/arduino/arduino-cli.git",
      :tag      => "0.11.0",
      :revision => "0296f4df116385f868b67c5ffa7393936c3345c9"
-  revision 1
+  revision 2
   head "https://github.com/arduino/arduino-cli.git"
 
   bottle do
     cellar :any_skip_relocation
     sha256 "3dd10065c03827d1cc0a508daaf2787c154eda4bfc5378d671b17d16672bd293" => :catalina
     sha256 "4d176c41402ad7a3ceb8ad20ef4a621675c20bdb080b543193efddad07e7fc26" => :mojave
     sha256 "d885f46abbb996e64a47d8566677263bca7ca2cacfe78d8bd825d5a4bba02f38" => :high_sierra
+    sha256 "2444faa21e3a470284d96cfef9667ff630fb39da7c7f619df21c6f0a68a671ac" => :x86_64_linux
```

Comment:
Do not modify the bottle block, it is handled automatically by our CI

Reply:
Removed, sorry